packages/renderer/src/lib/container/ContainerActions.svelte

```diff
@@ -173,4 +176,9 @@ if (dropdownMenu) {
     menu="{dropdownMenu}"
     detailed="{detailed}"
     icon="{faArrowsRotate}" />
+  <ContributionActions
+    args="{[{ ...container, icon: undefined }]}"
```

Comment:
Same as previous comment, the icon property is not serializable, therefore cannot be cloned.

Reply:
I think we could opt-out non-serialized arguments later just when we need to remove them

Reply:
here, we just give `container` object